topo_processor/util/checksum.py

```diff
@@ -5,7 +5,7 @@
 CHUNK_SIZE = 1024 * 1024  # 1MB
 
 
-def multihash_as_hex(file_path: str) -> str:
+async def multihash_as_hex(file_path: str) -> str:
```

Comment:
by just adding `async` does actually make this async.

you will need to use a async file reader to actually read from the disk in async manner

```
with aiofiles.open(file_path, 'rb' as file:
  while chunk := await file.read(CHUNK_SIZE)
```